index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.0.1/css/bootstrap.min.css">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body class="container">
+  <h1>Hello World, We are software developers!</h1>
+
+  <h1 id="title">Survery Form</h1>
+  <p id="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos quibusdam veniam alias est nam nihil
+    ipsa mollitia sapiente consectetur maiores placeat, recusandae illo facilis </p>
+  <form id="survey-form">
+    <label for="name" id="name-label">Name</label>
+    <input type="text" id="name" name="name" placeholder="Username" class="form-control mb-2" required>
+    <label for="email" id="email-label">E-mail</label>
+    <input type="email" id="email" name="email" placeholder="E-mail" class="form-control mb-2" required>
+    <label for="number" id="number-label">Number</label>
+    <input type="number" id="number" min="0" max="100" class="form-control mb-2" placeholder="age">
```

Comment:
I think it is better to reduce the with of the html tags to make them responsive on mobile